src/repology/repology_cli.py

```diff
@@ -90,6 +96,8 @@ def getargs():
     optional.add_argument("--verbose", help=helps, type=int, default=1)
     helps = "Path to output report file (default: ./repology_report.csv)"
     optional.add_argument("--out", help=helps, default="repology_report.csv")
+    if str_args:
+        return parser.parse_args(str_args.split())
```

Comment:
just `args`, without the spliting.

Reply:
Fixed in https://github.com/tiiuae/sbomnix/commit/ecf9b87d14ceb3b3668dc5b05746fc2b754d8195